server/views/pages/error.njk

```diff
@@ -1,7 +1,7 @@
 {% extends "../partials/layout.njk" %}
 
 {% set pageId = 'service-unavailable' %}
-{% set pageTitle = applicationName + " - Service unavailable" %}
+{% set pageTitle =  "Sorry, there is a problem with this service" %}
```

Comment:
Not that I have a problem with them, but where did we get all these page `<title>` element values from? Are they in some content document somewhere? because they don't appear to be listed or detailed in the Jira ticket at all?

Reply:
There were a number of key pages listed in the JIRA ticket, however pages like this must have been forgotten. So this example is just ensuring the page name `<h1>` is correct in the `<title>`